main.go

```diff
@@ -13,18 +12,14 @@ import (
 )
 
 func main() {
-	var fileName string
-	flag.StringVar(&fileName, "f", "", "YAML file to parse.")
-	flag.Parse()
-
-	if fileName == "" {
-		fmt.Println("Please specify your config file using the -f option")
-		return
+	home, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatal(err)
 	}
-
-	f, err := os.Open(fileName)
+	defaultConfigPath := fmt.Sprintf("%s/.config/gobee/config.yaml", home)
+	f, err := os.Open(defaultConfigPath)
```

Comment:
You could use `ioutil.ReadFile()` and get slice of bytes (`byte[]`) and later on `yaml.Unmarshal(contents, target)`. Just an alternative this is fine as well.

Reply:
`os.Open()` appears more succinct, is there more overhead with this function than `ioutil.ReadFile()`?

Reply:
Or does it mean that I would swap
```
	decoder := yaml.NewDecoder(f)
	err = decoder.Decode(&cfg)
```
for
```
yaml.Unmarshal(contents, target)
```
?

Reply:
From code I read `os.Open` is mostly used if there is more thing s to do with the file. `ioutil.ReadFile` is wrapper around it and less verbose in the end.

Reply:
Something like:

```go
	configContents, err := ioutil.ReadFile("../ca.pem")
	if err != nil {
		log.Fatalf("%s\nHave you created a Gobee config file at %s?", err, defaultConfigPath)
	}

	var cfg config.Config
	err = yaml.Unmarshal(configContents, &cfg)
	if err != nil {
		log.Fatalf("cmd.Run() failed with %s\n", err)
	}
```

Reply:
Like it. 
Done.

Reply:
The config is not going to be megabytes long no need to stream it; that's another reason.